x-pack/test/fleet_api_integration/config.base.ts

```diff
@@ -84,6 +84,8 @@ export default async function ({ readConfigFile }: FtrConfigProviderContext) {
             appenders: ['default'],
           },
         ])}`,
+        // Don't enforce versions header validation
+        `--server.versioned.versionResolution=oldest`,
```

Comment:
with this we don't have to provide version in all integration tests?

Reply:
I'm removing it, I think it's not needed anymore.